app/views/tasks/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<h2>This is edit page</h2>
+
+<%= form_with model: @task, class: 'edit-task' do |f| %>
+<p>Please make changes:</p>
+
+<div>
+    <%= f.label :name %>
+    <%= f.text_field :name %>
+</div>
+
+<div>
+    <%= f.label :description %>
+    <%= f.text_field :description %>
+</div>
+
+<div>
+    <%= f.label :completed_at %>
+    <%= f.text_field :completed_at %>
+</div>
```

Comment:
Consider removing this from the form so that it is just set with the `mark_complete` method.